input/resources/OperationDefinition-evaluate.xml

```diff
@@ -106,6 +94,23 @@
         <documentation value="The measure to be evaluated is provided as a Measure resource."/>
         <type value="Measure"/>
     </parameter>
+    <parameter>
+        <name value="programIdentifier"/>
+        <use value="in"/>
+        <min value="0"/>
+        <max value="1"/>
+        <documentation value="The business identifier of the reporting program, for which measures are being evaluated. For example, downstream IGs could specify program identifiers. Mutually exclusive use with programUrl - it is an error to pass in both programIdentifier and programUrl parameters."/>
+        <type value="string"/>
+        <searchType value="token"/>
+    </parameter>
+    <parameter>
+        <name value="programUrl"/>
+        <use value="in"/>
+        <min value="0"/>
+        <max value="1"/>
+        <documentation value="Mutually exclusive use with programUrl - it is an error to pass in both programIdentifier and programUrl parameters."/>
```

Comment:
It looks like the resolution has typo. The documentation here for programUrl should be "Mutually exclusive use with programIdentifier .... " instead of programUrl.

Reply:
Fixed in commit [36c1917](https://github.com/HL7/davinci-deqm/pull/225/commits/36c1917294c9e19a0f0f83ee3a6d83c08d308b94)